packages/drivers/file-driver/package.json

```diff
@@ -80,7 +80,7 @@
 		"@fluidframework/build-common": "^2.0.3",
 		"@fluidframework/build-tools": "^0.40.0",
 		"@fluidframework/eslint-config-fluid": "^5.3.0",
-		"@fluidframework/file-driver-previous": "npm:@fluidframework/file-driver@2.0.0-rc.5.0.0",
+		"@fluidframework/file-driver-previous": "npm:@fluidframework/file-driver@2.0.0",
```

Comment:
Noting for follow up - find the commit which would have added the "broken" entries.